src/aruku/walking/node/walking_node.cpp

```diff
@@ -124,7 +129,7 @@ void WalkingNode::publish_joints()
 
   joints_msg.control_type = tachimawari::joint::Middleware::FOR_WALKING;
 
-  set_joints_publisher->publish(joints_msg);
+  if (!disable_publish_joint) set_joints_publisher->publish(joints_msg);
```

Comment:
can put this condition on top of the function to return from the function to save some running time